addons/point_of_sale/static/tests/tours/pos_combo_tour.js

```diff
@@ -17,16 +17,8 @@ registry.category("web_tour.tours").add("PosComboPriceTaxIncludedTour", {
             ...ProductScreen.clickShowProductsMobile(),
             ...ProductScreen.clickDisplayedProduct("Office Combo"),
             combo.select("Combo Product 3"),
-            {
-                content: "check that amount is not displayed if zero",
-                trigger: `article.product .product-content:not(:has(.price-tag:contains("0")))`,
-                isCheck: true,
-            },
-            {
-                content: "check that amount is properly displayed when it is not 0",
-                trigger: `article.product .product-content .product-name:contains("Combo Product 3") ~.price-tag:contains("2.60")`,
-                isCheck: true,
-            },
+            combo.isAmountNotDisplayOnZero(),
+            combo.isAmountDisplay("Combo Product 3", "2.60"),
```

Comment:
restore this. If the step is not reused, we don't need to create a function for it. don't forget to remove the function in combo_popup_util. Someone who will introduce new tests on combo product workflow will introduce these utility methods.